include/tscore/ink_memory.h

```diff
@@ -103,9 +103,10 @@ int ats_mlock(caddr_t addr, size_t len);
 
 void *ats_track_malloc(size_t size, uint64_t *stat);
 void *ats_track_realloc(void *ptr, size_t size, uint64_t *alloc_stat, uint64_t *free_stat);
-void ats_track_free(void *ptr, uint64_t *stat);
+void  ats_track_free(void *ptr, uint64_t *stat);
 
-static inline size_t __attribute__((const)) ats_pagesize()
+static inline size_t __attribute__((const))
+ats_pagesize()
```

Comment:
This seems pretyy rando.

Reply:
Yah, it seems clang-format v18 actually fixed this where the function ought to start where it does. So, this is correct, and the previous clang-format was wrong.